### PR TITLE
Added note with more detail for # key to create-a-phone-system-auto-attendant.md

### PR DESCRIPTION
**Pull request by dariomws**
Closes https://github.com/MicrosoftDocs/OfficeDocs-SkypeForBusiness/issues/8139.

More information:
https://github.com/MicrosoftDocs/OfficeDocs-SkypeForBusiness/issues/5206

@CarolynRowe, could you kindly confirm? Thanks!

### CODE REVIEW

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: Teams/create-a-phone-system-auto-attendant.md [hunk 1]
```diff
@@ -98,6 +98,9 @@ If you select **Play menu options**, you can choose to **Play an audio file** or
 
 For dialing options, assign the 0-9 keys on the telephone keypad to one of the call routing destinations. (The keys \* (Repeat) and \# (Back) are reserved by the system and can't be reassigned.)
 
+> [!NOTE]
+> The # key only backs up to the most recent auto attendant, once the boundary is crossed to a new auto attendant, the # key will not be able to take you to the previous one.
```
Comment (JohanFreelancer9) [inline Teams/create-a-phone-system-auto-attendant.md:102]:
```suggestion
> The # key only backs up to the most recent auto attendant. Once the boundary is crossed to a new auto attendant, the # key will not be able to take you to the previous one.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - minor suggestion.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by scanum** [comment]
@CarolynRowe PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Pull request merged**